pkg/virt-handler/node-labeller/node_labeller.go

```diff
@@ -76,7 +76,7 @@ type NodeLabeller struct {
 	cpuCounter              *libvirtxml.CapsHostCPUCounter
 	hostCPUModel            hostCPUModel
 	SEV                     SEVConfiguration
-	arch                    string
+	Arch                    archLabeller
```

Comment:
why is this public?

Reply:
It is used in `virt-handler.go`.

I think you can create a method for the `NodeLabeller` called `ShouldLabelNodes` that uses `arch` and does not require to expose it.

Reply:
Sorry for the late update, i had meetings this afternoon.
I have moved arch to be private and exposed `ShouldLabelNodes()` via the node-labeller.